alicloud/resource_alicloud_route_table_association_test.go

```diff
@@ -0,0 +1,174 @@
+package alicloud
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"testing"
+
+	"github.com/aliyun/alibaba-cloud-sdk-go/sdk/requests"
+
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/vpc"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func init() {
+	resource.AddTestSweepers("alicloud_route_table_association", &resource.Sweeper{
+		Name: "alicloud_route_table_association",
+		F:    testSweepRouteTableAssociation,
+		// When implemented, these should be removed firstly
+		Dependencies: []string{},
+	})
+}
+
+func testSweepRouteTableAssociation(region string) error {
```

Comment:
I think this sweeper is a dependence of vswitch and route talbe's sweeper. Please check it.